### ISSUE TITLE
Icon request: icon-dialpad [ kinda Dial pad on phone ] (numeric pad, keypad)

### ISSUE THREAD

**Issue by arunkp** [FortAwesome/Font-Awesome#6266]
It would be awesome we can have a dialpad like icon
![image](https://cloud.githubusercontent.com/assets/4560500/7223977/531f6d0e-e749-11e4-8c12-4a622b7d51bb.png)

**Comment by alexander-semikin** [comment]
+1

**Comment by AlexandreFournier** [comment]
+1

**Comment by SylvainBILLIEMAZ9** [comment]
+1

**Comment by appjitsu** [comment]
+1

**Comment by nofxx** [comment]
Please add this. Use the new braille icon as kickstart.

**Comment by internalfx** [comment]
Yes, please

**Comment by aleyrizvi** [comment]
+1

**Comment by lwshon** [comment]
kinda shocked this isn't an option yet.

**Comment by guy032** [comment]
Whoever in need can find this in Google Material-Icons
[https://www.w3schools.com/icons/icons_reference.asp](https://www.w3schools.com/icons/icons_reference.asp)

**Comment by chrismilne-co** [comment]
+1

**Comment by yasinosman** [comment]
+1

**Comment by louis123562** [comment]
+1

**Comment by MarcoWa09** [comment]
+1

**Comment by drewgeraets** [comment]
+1

**Comment by alexgurr** [comment]
Anyone that's using Font Awesome pro, here's an SVG I made using the `braille` icon. Only `regular` for now though.

You can go into your kit and just upload it as `fa-dialpad`.


![dialpad-regular](https://user-images.githubusercontent.com/4161867/157136333-bef11b0d-845a-4a6a-a61f-908bb671c683.svg)

**Comment by lucholalev** [comment]
is there no icon for that yet?
would be nice to make a set of login options: dial pad for pin code, face for face recognition... etc
:)

**Comment by DesignGremlin** [comment]
Yes please! Here in 2023 wanting this icon to indicate switching between the number keyboard to the standard keyboard and back.

**Comment by saropa** [comment]
This icon would be really useful right now!

**Comment by AachoLoya** [comment]
This request has been pending since 2015 and icon is very simple too. Can you please create this icon now?

**Comment by cmarker0** [comment]
+1

**Comment by arunkp** [comment]
Well, 2 more years and my daughter will raise a PR.

**Comment by saropa** [comment]
Looking at the PRs, the main focus seems to be brand logos

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by AachoLoya** [comment]
Its been a year since I last posted, 9 years since original request. How come such a simple icon hasn't been created yet?

**Comment by mehdika2** [comment]
+1

**Comment by VadimGouskov** [comment]
+1

**Comment by laurielabergepetalmd** [comment]
+1

**Comment by saropa** [comment]
This would be very useful — its the last Material icon we need replacing in our app

**Comment by gem85247** [comment]
Any update on this (timeline wise)? thanks in advance!

**Comment by sensibleworld** [comment]
Yup! This will be in version 7, launching next month!

**Comment by sensibleworld** [comment]
It only took us a decade! https://fontawesome.com/icons/dialpad?f=classic&s=solid
